docs/web/index.md

```diff
@@ -577,18 +577,18 @@ Services that use the sda-spring-boot-commons:
 - are configurable by the property or environment variable `METADATA_FIELDS` to be aware of the
   metadata used in a specific environment
 
-Services that interrupt a business process should persist the context from
-`MetadataContext.detachedCurrent()` and restore it with `MetadataContext.createContext(…)` when the
-process continues.
+### Async processes
+
```

Comment:
Imho, we should keep the introduction. The first part of these paragraphs is about interrupting in terms of persisting. The last paragraph is about async (line 591+ in the old version).